docs/user-guide/ingest-data/for-observability/elasticsearch.md

```diff
@@ -11,7 +11,9 @@ GreptimeDB supports data ingestion through Elasticsearch's [`_bulk` API](https:/
 
 ## HTTP API
 
-GreptimeDB supports data ingestion via the Elasticsearch protocol through two HTTP endpoints:
+In most log collectors(such as Logstash and Filebeat mentioned below), you only need to configure the HTTP endpoint as `/v1/elasticsearch`, for example `http://localhost:4000/v1/elasticsearch`.
```

Comment:
Personally, I prefer `http://{db_host}:4000/v1/elasticsearch` as the sample URL. WDYT @nicecui

Reply:
LGTM